### PR TITLE
chore: (README.md) Update readme to add bugs, fr

### PR DESCRIPTION
**Pull request by sammartinez**
_Issue #, if available:_

_Description of changes:_
With the majority of our issues being related to feature requests, I wanted to provide a way to customers to see this on our README. This will show our feature requests currently present along with current bugs and closed issues

Screenshot:
feature_requests_showing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,11 @@
   </a>
 </p>
 
+## Reporting Bugs/Feature Requests
```
Comment (ericclemmons) [inline README.md:20]: I like these stats!  They really help indicate the "health" better than the 658 at the top of the page.

Is this headline needed?  It feels a bit out-of-place at the top of the README.

(CC @renebrandel)

Comment (sammartinez) [reply README.md:20]: This is similar to what iOS is doing so Im curious what @renebrandel thinks as well. May not be needed but I can make the update as needed

**Review by ashika01 — APPROVED** [review]
I like this 👍 Maybe bring down the header size by 1?

**Comment by renebrandel** [comment]
Love it. We should also add this to the CLI repo.

**Comment by sammartinez** [comment]
> Love it. We should also add this to the CLI repo.

Sounds good! Ill look to cut a PR to CLI today then for it :)

File: README.md (hunk 1 above)
Comment (ericclemmons) [inline README.md:20]: @sammartinez  You mentioned turning this into an H3, so I made it a cinch for you to merge :D

```suggestion
### Reporting Bugs/Feature Requests
```

**Review by ericclemmons — APPROVED** [review]
Ready to approve, but I added @sammartinez 's change to an H#.

**Pull request merged**